charts/ztka/values.yaml

```diff
@@ -296,6 +296,16 @@ kratos:
               oidc:
                 hooks:
                   - hook: session
+          login:
+            ui_url: http://dashboard:3000/login
+            after:
+              password:
+                hooks:
+                  - hook: web_hook
+                    config:
+                      url: http://paralus:11000/auth/v3/user/auditlog
```

Comment:
do not hardcode the port, user can override it using services block in values.yaml

Reply:
completed